lib/experimental/Information/Shortcut/index.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import type { Meta, StoryObj } from "@storybook/react"
+import { ComponentProps } from "react"
+import { Shortcut } from "."
+
+const meta = {
+  component: Shortcut,
+  parameters: {
+    layout: "centered",
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/design/pZzg1KTe9lpKTSGPUZa8OJ/Web-Components?node-id=172-2793",
+    },
+  },
+  tags: ["autodocs"],
+  args: {
+    keys: ["cmd", "k"],
+  } satisfies ComponentProps<typeof Shortcut>,
+} satisfies Meta<typeof Shortcut>
+
+export default meta
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {}
+
+export const MultipleKeys: Story = {
+  args: {
+    keys: ["↑", "↑", "↓", "↓", "←", "→", "←", "→", "B", "A"],
+  },
+}
```

Comment:
A couple more examples wouldn't hurt. What if there's no horizontal space and you have multiple keystroke elements? How does it wrap?

I think we can get used to adding examples in context.